Civi/Api4/Query/SqlFunctionDATE_SUB.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+namespace Civi\Api4\Query;
+
+/**
+ * Sql function
+ */
+class SqlFunctionDATE_SUB extends SqlFunction {
+
+  protected static $category = self::CATEGORY_DATE;
+
+  protected static function params(): array {
+    return [
+      [
+        'label' => ts('The value of the time/date interval to subtract.'),
+        'optional' => FALSE,
+        'must_be' => ['SqlNumber'],
```

Comment:
Shouldn't this allow fields as well as literal numbers?

Reply:
Interval is actually, positive or negative number.

Reply:
Oh ok but then this is missing a param. Should accept the field parm before this one, right?

Reply:
yes,  I have updated the params now.